proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -388,4 +390,20 @@ message StreamWorkflowReplicationMessagesResponse {
     oneof attributes {
         temporal.server.api.replication.v1.WorkflowReplicationMessages messages = 1;
     }
+}
+
+message GetNamespaceRequest {
+    oneof attributes {
+        string namespace = 1;
```

Comment:
nit: just name (compare to the 2nd field id)

Reply:
Yeah, I was planing to use `name` as well, but protobuf complains "https://linter.aip.dev/131/request-name-behavior" this error. I think that's probably the reason that namespace is used [here](https://github.com/temporalio/api/blob/f229641c576b421ac16b4ac73f35f1bf083c6572/temporal/api/workflowservice/v1/request_response.proto#L97)